apps/sertra/components/navbar.tsx

```diff
@@ -0,0 +1,41 @@
+import { FC, PropsWithChildren } from "react";
+
+export const Navbar: FC<PropsWithChildren> = () => {
+  return (
+    <div style={{ width: "1140px", marginLeft: "auto", marginRight: "auto" }}>
```

Comment:
I was going to suggest that this could use `TopBar` from `@thunderstore/components`, but it seems it's not that easy, **maybe** due to React version mismatch. Maybe not an issue for MVP, but in the long run we probably want to be able to use shared components here?

Reply:
Yeah possibly, at this stage I'm intentionally keeping them separate because the `@thunderstore/components` library is under more scrutiny than this app, so we could end up inadvertently slowing down the development velocity here by relying on the shared components.

We'll just need to make a conscious effort in consolidating common elements to the component library once it has stabilized a bit.